server/books/models.py

```diff
@@ -1,3 +1,140 @@
 from django.db import models
+from django.contrib.auth import get_user_model
 
-# Create your models here.
+
+User = get_user_model()
+
+
+class Book(models.Model):
+    """
+    Book Model
+
+    Fields
+    ----------
+    title : CharField
+        Title of book
+    author : CharField
+        Book author
+    page_count : CharField
```

Comment:
There's a small typo in the comment here

```suggestion
    page_count : PositiveSmallInteger
```